src/bot.ml

```diff
@@ -192,6 +192,63 @@ let callback _conn req body =
             |> Lwt.async ;
             Server.respond_string ~status:`OK ~body:"Handling minimization." ()
             )
+          else if
+            string_match
+              ~regexp:
+                (f "@%s:? [Cc][Ii][- ][Mm]inimize:?\\( [^\n]*\\)?" bot_name)
+              body
+          then (
+            let requests =
+              String.split ~on:' '
+                (Stdlib.String.trim
+                   (* FIXME why not `String.trim`? ocamlc complains it's unbound... (cf https://coq.zulipchat.com/#narrow/stream/243318-coqbot-devs.20.26.20users/topic/.60String.2Etrim.60/near/236752334) *)
```

Comment:
Yes, this is a function from the stdlib, but the modules from the stdlib are shadowed by the ones from the Base library.

Reply:
Okay, I've removed the comment